homeassistant/components/nam/strings.json

```diff
@@ -39,15 +39,125 @@
     }
   },
   "entity": {
+    "button": {
+      "restart": {
+        "name": "[%key:component::button::entity_component::restart::name%]"
+      }
+    },
     "sensor": {
-      "caqi_level": {
+      "bme280_humidity": {
+        "name": "BME280 [%key:component::sensor::entity_component::humidity::name%]"
+      },
+      "bme280_pressure": {
+        "name": "BME280 [%key:component::sensor::entity_component::pressure::name%]"
+      },
+      "bme280_temperature": {
+        "name": "BME280 [%key:component::sensor::entity_component::temperature::name%]"
+      },
+      "bmp180_pressure": {
+        "name": "BMP180 [%key:component::sensor::entity_component::pressure::name%]"
+      },
+      "bmp180_temperature": {
+        "name": "BMP180 [%key:component::sensor::entity_component::temperature::name%]"
+      },
+      "bmp280_pressure": {
+        "name": "BMP280 [%key:component::sensor::entity_component::pressure::name%]"
+      },
+      "bmp280_temperature": {
+        "name": "BMP280 [%key:component::sensor::entity_component::temperature::name%]"
+      },
+      "heca_humidity": {
+        "name": "HECA [%key:component::sensor::entity_component::humidity::name%]"
+      },
+      "heca_temperature": {
+        "name": "HECA [%key:component::sensor::entity_component::temperature::name%]"
+      },
+      "mhz14a_carbon_dioxide": {
+        "name": "MH-Z14A [%key:component::sensor::entity_component::carbon_dioxide::name%]"
+      },
+      "pmsx003_caqi": {
+        "name": "PMSx003 Common air quality index"
+      },
+      "pmsx003_caqi_level": {
+        "name": "PMSx003 Common air quality index level",
```

Comment:
I used a name similar to AQI
https://github.com/home-assistant/core/blob/580b20b0a83c561986e7571b83df4a4bcb158392/homeassistant/components/sensor/strings.json#L131